internal/proxy/impl.go

```diff
@@ -3647,10 +3648,35 @@ func (node *Proxy) Import(ctx context.Context, req *milvuspb.ImportRequest) (*mi
 		resp.Status.Reason = err.Error()
 		return resp, nil
 	}
+	log.Info("import complete, now completing import", zap.Int64s("task IDs", respFromRC.GetTasks()))
+	for _, taskID := range respFromRC.GetTasks() {
+		go node.completeImport(taskID)
+	}
 	return respFromRC, nil
 }
 
-// GetImportState checks import task state from datanode
+func (node *Proxy) completeImport(taskID int64) {
+	// First check if the import task has turned into persisted state. Returns an error status if not after retrying.
+	// This could take a few or tens of seconds.
+	getImportResp, err := node.checkImportTaskPersisted(taskID)
+	if err != nil {
+		log.Error("task not persisted yet after wait limit",
+			zap.Int64("wait limit (seconds)", int64(CheckTaskPersistedWaitLimit.Seconds())),
+			zap.Int64("task ID", taskID),
+			zap.Any("current task state", getImportResp.GetState()))
+		return
+	}
+
+	// Start background process of complete bulk load operation.
+	// Ignoring complete bulk load return status.
+	node.dataCoord.CompleteBulkLoad(node.ctx, &datapb.CompleteBulkLoadRequest{
```

Comment:
Doesn't make sense to me.
If impormanager is at Rootcoord, why proxy go to datacoord directly to complete Bulkload?
What if proxy crashed?